src/archivist/recorder/repositories/git/git.js

```diff
@@ -32,35 +32,44 @@ export default class Git {
     return this.git.add(this.relativePath(filepath));
   }
 
-  async commit(filepath, message, authorDate) {
-    const options = {};
+  async commit({ filepath, message, date = new Date() }) {
+    let summary;
 
-    if (authorDate) {
-      options['--date'] = new Date(authorDate).toISOString();
-    }
+    try {
+      const commitDate = new Date(date).toISOString();
```

Comment:
I not really agree. As the finally is here to rollback env setup even in case of error, I prefer to have the `try` wrapping the content related to the `finally`.